record-editor/src/app/backoffice-editor/backoffice-editor.component.ts

```diff
@@ -86,8 +86,16 @@ export class BackofficeEditorComponent
     }
   }
 
+  private injectValidationErrorsIntoExtraData() {
+    this.workflowObject._extra_data.validation_errors = this.workflowObject.validation_errors;
+  }
+
   get workflowExtraData() {
-    return this.workflowObject._extra_data || {};
+    this.workflowObject._extra_data = this.workflowObject._extra_data || {};
+    if (this.workflowObject.validation_errors) {
```

Comment:
why do we now require this?

Reply:
As described in https://github.com/cern-sis/issues-inspire/issues/585, the plan is to add the `validation_errors` field into the backend response (because we dont want to have this `extra_data` field anymore). But as we dont want to change much in the record editor (as it need to work with next and backoffice records), we inject the `validation_errors` field into the `extra_data` field